From: SeongJae Park <sj@kernel.org>
Date: Sat, 26 Jul 2025 13:44:57 -0700
Subject: [PATCH] mm/damon/core: remove max_avail_nr_accesses in
 kdamond_apply_access_report()

It is not really being used.  Remove.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/damon/core.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/mm/damon/core.c b/mm/damon/core.c
index 4f684ec7a200..428ac1b83118 100644
--- a/mm/damon/core.c
+++ b/mm/damon/core.c
@@ -2565,14 +2565,10 @@ static void kdamond_apply_access_report(struct damon_access_report *report,
 		struct damon_target *t, struct damon_ctx *ctx)
 {
 	struct damon_region *r;
-	unsigned int max_avail_nr_accesses;
 
 	if (ctx->ops.eligible_report && !ctx->ops.eligible_report(report, t))
 		return;
 
-	max_avail_nr_accesses = ctx->attrs.aggr_interval /
-		ctx->attrs.sample_interval;
-
 	/* todo: make search faster, e.g., binary search? */
 	damon_for_each_region(r, t) {
 		if (report->addr < r->ar.start)
-- 
2.39.5

